ani-cli

```diff
@@ -198,53 +198,48 @@ get_video_link() {
 get_video_quality_mp4() {
 	case $quality in
 		best)
-			video_url=$(printf '%s' "$1" | head -n 4 | tail -n 1) ;;
+			video_url=$(select_line "$1" 4) ;;
 		worst)
-			video_url=$(printf '%s' "$1" | head -n 1) ;;
+			video_url=$(select_line "$1" q1) ;;
 		*)
 			video_url=$(printf '%s' "$1" | grep -i "${quality}p" | head -n 1)
 			if [ -z "$video_url" ]; then
 				err "Current video quality is not available (defaulting to best quality)"
 				quality=best
-				video_url=$(printf '%s' "$1" | head -n 4 | tail -n 1)
+				video_url=$(select_line "$1" 4)
 			fi
 			;;
 	esac
 	printf '%s' "$video_url"
 }
 
+# select the $2th line in $1
+select_line() {
+	printf '%s' "$1" | head -n "$2" | tail -n 1
+}
```

Comment:
Should this stay? This is something we tend to use more and more often, but it's a single line, so could very well be inline...